content/about.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "About"
+date: 2017-07-03T14:41:50-04:00
+draft: false
+mainpage: false
+projectpage: false
+projectslug: ""
+projectname: ""
+listdescription: ""
+---
+
+### About
+
+WorldWind is a free, open source API for visualizing and hosting geospatial data. Our client libraries, available for Android devices, the browser, or the desktop, provide developers a quick and easy utility for visualizations of 3D globe, map and geographical information. Our server kit extends the functionality of GeoServer to provide an end to end geospatial solution.
+
+<img align="left" src="/img/800px-worldwind_screen.jpg" width="30%" hspace="10" vspace="10"> WorldWind Organizations across the world use WorldWind to monitor weather patterns, visualize cities and terrain, track the movement of planes, vehicles and ships, analyze geospatial data, and educate people about the Earth.
```

Comment:
I generated new size appropriate screen shots from WebWW and WWA. I added the 'img-responsive' class, which will shrink the images as the screen size reduces, but since the images are starting at more reasonable resolutions, I don't think the distortion is too bad.